.github/workflows/nightly-playground-deploy.yml

```diff
@@ -77,6 +77,7 @@ jobs:
           npm install
           playground_id=`echo ${{inputs.dist_version}} | cut -d. -f1`x
           echo "PLAYGROUND_ID=$playground_id" >> "$GITHUB_OUTPUT"
+          aws s3 cp s3://nightly-playgrounds-snapshots-bucket/internal_users.yml resources/security-config/internal_users.yml
```

Comment:
I assume this bucket is already pre-created and has the `internal_users.yml` stored, does this file gets updated for every new version ? or we maintain one standard file for all version?

Reply:
One standard for all versions. Yes its pre-created via https://github.com/opensearch-project/opensearch-devops/blob/main/nightly-playground/lib/common-tools-stack.ts#L45